lib/src/api.dart

```diff
@@ -95,6 +96,21 @@ class API {
     return _responseHandler(response);
   }
 
+  Future<String> login(String token, String accountAddress) async {
+    Map<String, dynamic> resp = await _post('v2/login', body: {
+      "token": token,
```

Comment:
maybe rename token to "firebaseToken" to make to it more precise?

Reply:
you missed the comments train :) already approved,  merged and published 😆

Reply:
lol ><